packages/vuetify/src/composables/list-items.ts

```diff
@@ -63,7 +63,7 @@ export function transformItem (props: Omit<ItemProps, 'items'>, item: any): List
   const itemProps = props.itemProps === true
     ? typeof item === 'object' && item != null && !Array.isArray(item)
       ? 'children' in item
-        ? omit(item, ['children'])
+        ? omit(item as any, ['children'])
```

Comment:
This use of "any" here is probably a red flag for @KaelWD

Reply:
I can remove it, I think, since `transformItem` already types it as `any`.